src/styles/blocks/reset.scss

```
@@ -0,0 +1,25 @@
+*,
+*::after {
```

This is redundant, * works for after anyway
```suggestion
* {
```